libs/eo/shared/atomic-design/ui-atoms/src/lib/eo-pie-chart/eo-pie-chart.component.ts

```diff
@@ -0,0 +1,82 @@
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Component, NgModule } from '@angular/core';
+import { ChartConfiguration, ChartData, ChartType } from 'chart.js';
+import { NgChartsModule } from 'ng2-charts';
+import DatalabelsPlugin from 'chartjs-plugin-datalabels';
+
+@Component({
+  selector: 'eo-pie-chart',
+  template: `<canvas
+    baseChart
+    [data]="pieChartData"
+    [type]="pieChartType"
+    [options]="pieChartOptions"
+    [plugins]="pieChartPlugins"
+  >
+  </canvas>`,
+  styles: [``],
+})
```

Comment:
fix/question:

Since this is state of the component, we could consider moving this part into a ComponentStore. It might blow up the task, so talk with Lars on how we should do this for now.

See example of state management inside, i.eg. 

- eo-privacy-policy.store.ts
- eo-media.presenter.ts

Reply:
My own view on this, would be that i would rather manage this components state at component-level :) I was thinking we could reuse the component around the app by providing inputs for the colors, labels and so on.

Reply:
It's not necessary for now but the need will increase if we combine data sources or add more complex business logic. Let's deal with it if something like that happens. I have real world examples of splitting chart concerns into presenters.

Reply:
Sure, I think as long as state management is somewhat consistent across the app, we are good - But Component Store is actually meant to be on component level.

Reply:
Will skip this for now then, but bear it in mind for when the chart is gonna become more generic :) 

Thanks both!